.changeset/rotten-ghosts-prove.md

```diff
@@ -0,0 +1,9 @@
+---
+"@logto/next-app-dir-sample": minor
+"@logto/next-sample": minor
+"@logto/client": minor
+"@logto/node": minor
+"@logto/js": minor
+---
+
+Add RBAC support for NextJS SDK
```

Comment:
do we support RBAC in other frameworks like React? cc @charIeszhao

Reply:
I have an issue LOG-6138 to track this.